packages/create-plugin/src/constants.ts

```diff
@@ -86,6 +86,8 @@ export const MIGRATION_CONFIG = {
 export const UDPATE_CONFIG = {
   // Files that should be overriden between configuration version updates.
   filesToOverride: ['.config/', '.cprc.json'],
+  // Files that are no longer needed for the project and can be removed.
+  filesToRemove: ['./config/webpack/publicPath.ts'],
```

Comment:
As a conversation starter: shouldn't we compare what's not existing amongst the new template files for the `.config/` folder and what's present in the CWD, and delete the difference instead?

Reply:
I did consider trashing the .config directory and recreating it from the templates (which I believe would have the same outcome as you're suggesting). For now felt it was easier to reuse what we have from the migrate script.